Userland/Libraries/LibJS/Runtime/Completion.cpp

```diff
@@ -39,8 +39,9 @@ ThrowCompletionOr<Value> await(VM& vm, Value value)
     // 2. Let promise be ? PromiseResolve(%Promise%, value).
     auto* promise_object = TRY(promise_resolve(vm, realm.intrinsics().promise_constructor(), value));
 
-    Optional<bool> success;
-    Value result;
+    DOES_NOT_ESCAPE Optional<bool> success;
+    DOES_NOT_ESCAPE Value result;
```

Comment:
It feels weird to say that locals don't escape, because they can never escape.
It's really the lambda that doesn't escape, not the locals.

If we have to mark them with something, perhaps we can find something more accurate, even if obnoxiously verbose? `IGNORE_USE_IN_ESCAPING_LAMBDA` or something like that?